indico/modules/events/editing/notifications.py

```diff
@@ -0,0 +1,84 @@
+# This file is part of Indico.
+# Copyright (C) 2002 - 2020 CERN
+#
+# Indico is free software; you can redistribute it and/or
+# modify it under the terms of the MIT License; see the
+# LICENSE file for more details.
+
+from flask import session
+
+from indico.core.notifications import make_email, send_email
+from indico.modules.events.editing.schemas import EditingConfirmationAction
+from indico.web.flask.templating import get_template_module
+
+
+def notify_submitter_comment(revision):
+    """
+    Notify the editor about comments made by a submitter.
+    """
+    submitter = revision.submitter
+    editor = revision.editable.editor
+    tpl = get_template_module('events/editing/emails/submitter_comment_notification.txt',
+                              submitter_name=submitter.first_name, timeline_url=revision.editable.external_timeline_url,
+                              recipient_name=editor.first_name)
+    send_email(make_email(editor.email, template=tpl))
+
+
+def notify_editor_comment(revision, admin=False):
```

Comment:
maybe `manager` instead of `admin` is more suitable here? while an admin is manager-like in this case, it's more likely to be a manager than an indico admin leaving a comment